gradle/libs.versions.toml

```diff
@@ -38,15 +38,16 @@ korio = "4.0.10"
 
 # Sample - Android
 androidx-core = "1.13.1"
-androidx-lifecycle = "2.8.4"
-androidx-activity-compose = "1.9.1"
-android-desugaring = "2.0.4"
+androidx-lifecycle = "2.8.6"
+androidx-activity-compose = "1.9.2"
+android-desugaring = "2.1.2"
```

Comment:
These could not be upgraded higher as it would be a conflict of versions with compose multiplatform